assignments/callbacks.js

```diff
@@ -6,43 +6,81 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
   //Given this problem: 
   
-  function firstItem(arr, cb) {
-    // firstItem passes the first item of the given array to the callback function.
-  }
-
-  // Potential Solution:
-  function firstItem(arr, cb) {
-    return cb(arr[0]);
-  }
-
-  firstItem(items, function(first) {
-    console.log(first)
-  });
-
-*/
+  
+  */
+ 
+ function firstItem(arr, cb) {
+   // firstItem passes the first item of the given array to the callback function.
+ }
 
+ // Potential Solution:
+ function firstItem(arr, cb) {
+   return cb(arr[0]);
+ }
 
+ firstItem(items, function(first) {
+   console.log(first)
+ });
+ 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+
+  console.log(cb(arr));
+}
+
+let arrLength = function (items) {
+  return items.length;
 }
+getLength(items, arrLength);
 
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+
+  return cb(arr[arr.length-1]);
 }
 
+last(items, function(lastItem) {
+  console.log(lastItem);
+});
+
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+ return cb(x, y);
 }
 
+sumNums(5, 10, function(x, y) {
+  console.log(x + y);
+});
+
 function multiplyNums(x, y, cb) {
   // multiplyNums multiplies two numbers and passes the result to the callback.
+  return cb(x, y);
 }
 
+multiplyNums(5, 10, function(x, y) {
+  console.log(x * y);
+});
+
+
 function contains(item, list, cb) {
   // contains checks if an item is present inside of the given array/list.
   // Pass true to the callback if it is, otherwise pass false.
+  if(list.includes(item)){
+    return cb(true);
+  } else {
+    return cb(false);
+  }
 }
```

Comment:
```suggestion
function contains(item, list, cb) {
  // contains checks if an item is present inside of the given array/list.
  // Pass true to the callback if it is, otherwise pass false.
  for (let i = 0; i < list.length; i++) {
    if (list[i] === item) {
      return cb(true);
    }
  }
  return cb(false);
}
``` example solution